app/javascript/dashboard/components/widgets/mentions/MentionBox.vue

```diff
@@ -68,6 +68,7 @@ export default {
     },
     selectedIndex() {
       const container = this.$refs.mentionsListContainer;
+      if (!container) return;
```

Comment:
If `container` is not available, perhaps we can just search the ID on the `$root` or `$parent`

Also, did you manage to reproduce this? Add a watcher and see it's value, where in the lifecycle is it missing?

Reply:
@scmmishra I just wanted to clarify that the mentionsListContainer is the wrapper holding the list items. I've reproduced this issue. When we open canned responses in the editor using the `/` key and search for canned responses that are not available in the list then if we accidentally click the arrow keys in that case the container section is missing and throws an error.

Reply:
@scmmishra Done